be/src/runtime/stream_load/stream_load_context.h

```diff
@@ -197,6 +197,10 @@ class StreamLoadContext {
 
     int32_t idle_timeout_sec = -1;
 
+    // buf for reading data from ev_buffer
+    static const size_t buf_default_sz = 64 * 1024;
+    ByteBufferPtr buf = ByteBuffer::allocate(buf_default_sz);
```

Comment:
This allocation is useless since a new buffer will be allocated in `_on_header`, am I right?

Reply:
Only `CONTENT_LENGTH` is specified the buffer would be re-allocated. Maybe a better solution is to allocate memory when it is needed.